src/main/java/org/folio/dao/util/DbUtils.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.dao.util;
+
+import static org.folio.rest.persist.PostgresClient.convertToPsqlStandard;
+
+public class DbUtils {
```

Comment:
to avoid inheritance make it private class

Reply:
I believe I you have meant "final"

Reply:
right